BuildScript/BuildTargets/Testing.targets

```diff
@@ -228,6 +228,10 @@ $(_dockerImageName) $(_nunitCallArgumentString)
       <Output TaskParameter="Value" PropertyName="_testTime" />
     </MSBuild.ExtensionPack.Xml.XmlFile>
 
+    <PropertyGroup>
+      <_testTime Condition="'$(_testTime)' == ''">0</_testTime>
+    </PropertyGroup>
+
```

Comment:
Isn't this a separate issue?

Reply:
Funnily enough, no. This was the reason why the `NullReferenceException` occurred, the duration attribute was not defined in the result XML when no tests were found.

The other extra handling is to circumvent the fact that the XML is also marked as "Failed" if no tests are found.